tascrum/serializers.py

```diff
@@ -614,19 +653,32 @@ class Meta:
         fields = ['id', 'labels']
 
     def get_labels(self, obj):
-        label = obj.boardl.all()
-        return LabelTimelineSerializer(label, many=True).data
+        label = obj.boardl.filter(board=obj.id)
+        return LabelTimelineSerializer(label, many=True , context={'view': self.context.get('view')}).data
 
 class TimelineStartSerializer(serializers.ModelSerializer):
     class Meta:
         model = Card
         fields = ['startdate']
+    
```

Comment:
it'd better handled these details in frontend

Reply:
i checked with matin. he said NO.